public/locales/en-US/send.ftl

```diff
@@ -87,3 +87,4 @@ footerLinkCookies = Cookies
 requirePasswordCheckbox = Require a password to download this file
 addPasswordButton = Add password
 passwordTryAgain = Incorrect password. Try again.
+passwordResult = Password:
```

Comment:
Can you add a short localization note? e.g.

```
// This label is followed by the password set to download a file
```

Is it just me or #622 shows a completely different string?

Reply:
It does, though I believe the consensus was to change it to just be "password", though it's possible that's just what it became in the last iteration....

Reply:
https://github.com/mozilla/send/issues/381#issuecomment-327930546 has the mockups (last image in "sender" shows the string "Password required to download this file:")

Reply:
I double-checked with some folks, and the long one is preferred. Changing it, thanks!